alkos/kernel/arch/x86_64/abi/terminal/terminal.cpp

```diff
@@ -2,9 +2,9 @@
 #include <terminal.hpp>
 #include <defines.hpp>
 
-#ifdef __SERIAL_PORT_TEST__
```

Comment:
Why is printing functionality dependent on flags of a test?

Reply:
This flag simply indicates whether we are using serial port or not in the build. For example in release build there should be no writes to any port at all etc